src/controllers/playback/video/index.js

```diff
@@ -181,6 +181,14 @@ import { appRouter } from '../../../components/appRouter';
             } else {
                 view.querySelector('.btnAudio').classList.add('hide');
             }
+
+            if (playbackManager.currentItem(player).Chapters.length > 1) {
```

Comment:
No need to retrieve it again.
```suggestion
            if (currentItem.Chapters.length > 1) {
```